README.md

```diff
@@ -1,96 +1,183 @@
-import { documentRopstenValidWithToken } from "./test/fixtures/v2/documentRopstenValidWithToken";
-import { verify, isValid } from "@govtechsg/oa-verify";
+### Verifying a document
 
-const fragments = await verify(documentRopstenValidWithToken);
-console.log(fragments); // see below
-console.log(isValid(fragments)); // display true
-```
+A verification happens on a wrapped document, and it consists of answering to some questions:
+
+- Has the document been tampered with ?
+- Is the issuance state of the document valid ?
+- Is the document issuer identity valid ? (see [identity proof](https://www.openattestation.com/docs/docs-section/how-does-it-work/issuance-identity))
+
+A wrapped document (shown below) created using [Open Attestation](https://www.openattestation.com/docs/developer-section/libraries/open-attestation) would be required.
 
 ```json
-[
-  {
-    "data": true,
-    "name": "OpenAttestationHash",
-    "status": "VALID",
-    "type": "DOCUMENT_INTEGRITY"
-  },
-  {
-    "data": {
-      "details": [
-        {
-          "address": "0xe59877ac86c0310e9ddaeb627f42fdee5f793fbe",
-          "minted": true
+{
+  "version": "https://schema.openattestation.com/2.0/schema.json",
+  "data": {
+    "issuers": [
+      {
+        "documentStore": "746531fb-bcbf-44d1-a32f-d662c411a71e:string:0x8Fc57204c35fb9317D91285eF52D6b892EC08cD3",
+        "name": "824f1c2e-e289-4574-b207-d39afb151592:string:University of Blockchain",
+        "identityProof": {
+          "type": "e92275d8-5e8f-4adf-98fe-62e615f9837d:string:DNS-TXT",
+          "location": "3628440e-c859-4eec-bf47-fffedafec154:string:example.openattestation.com"
         }
-      ],
-      "mintedOnAll": true
-    },
-    "name": "OpenAttestationEthereumTokenRegistryStatus",
-    "status": "VALID",
-    "type": "DOCUMENT_STATUS"
+      }
+    ]
   },
-  {
-    "name": "OpenAttestationEthereumDocumentStoreStatus",
-    "reason": {
-      "code": 4,
-      "codeString": "SKIPPED",
-      "message": "Document issuers doesn't have \"documentStore\" or \"certificateStore\" property or DOCUMENT_STORE method"
-    },
-    "status": "SKIPPED",
-    "type": "DOCUMENT_STATUS"
+  "privacy": { "obfuscatedData": [] },
+  "signature": {
+    "type": "SHA3MerkleProof",
+    "targetHash": "0badef8f1d5652abef918c15725412b715c708d5eb25fe14df155d63c5241f62",
+    "proof": [],
+    "merkleRoot": "0badef8f1d5652abef918c15725412b715c708d5eb25fe14df155d63c5241f62"
+  }
+}
+```
+
+To perform verification check on the document:
+
+```javascript
+// index.js
+const { isValid, openAttestationVerifiers, verificationBuilder } = require("@govtechsg/oa-verify");
+const document = require("./document.json");
+
+const verify = verificationBuilder(openAttestationVerifiers, {
+  network: "ropsten",
+});
+
+verify(document).then((fragments) => {
+  console.log(isValid(fragments)); // output true
+});
+```
+
+### Custom verification
+
+By default the provided `verify` method performs multiple checks on a document
```

Comment:
this is not true anymore, there are missing verifiers

Reply:
ok, included the missing verifiers:

- for the type `DOCUMENT_STATUS`: it runs `OpenAttestationEthereumDocumentStoreStatus`, `OpenAttestationEthereumTokenRegistryStatus` and **`DidSignedDocumentStatus`** verifiers
- for the type `DOCUMENT_INTEGRITY`: it runs `OpenAttestationHash` verifier
- for the type `ISSUER_IDENTITY`: it runs `OpenAttestationDnsTxt`, **`DnsDidProof`** and **`DidIdentityProof`** verifiers

Reply:
`DidIdentityProof` is not included by default

Reply:
right, I overlooked it. removed it in the latest commit.